perf/common.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+exports.createMsg = function(msgType, sz) {
+    var buf;
```

Comment:
hey Michele, can we use double spaces for JS tabs, or basically half the tab size of this javascript? 

@nickdesaulniers, don't we want to implement that standard for JS?